include/jemalloc/jemalloc_macros.h.in

```diff
@@ -2,7 +2,11 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <limits.h>
-#include <strings.h>
+#ifdef _WIN32
+#  include "msvc_compat/strings.h"
```

Comment:
We have this include here: https://github.com/jemalloc/jemalloc/blob/dev/include/jemalloc/internal/jemalloc_internal_decls.h#L8

Shouldn't this be covered with that? Sorry I don't have the platform you are using so can't try it myself.